src/components/viewReview.jsx

```diff
@@ -0,0 +1,53 @@
+import { useParams, Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { fetchSingleReview } from "./api-requests";
+
+const ViewReview = () => {
+  const { review_id } = useParams();
+  const [loading, setLoading] = useState(true);
+  const [review, setReview] = useState({});
+
+  useEffect(() => {
+    setLoading(true);
+    fetchSingleReview(review_id).then((data) => {
+      setLoading(false);
+      return setReview(data.review);
+    });
+  }, []);
+
+  const handleClick = (e) => {};
+
+  return loading ? (
+    <h1>Loading..</h1>
```

Comment:
Consider using an h2 here. Your header uses an h1 and we only want one! This makes it easier for screen readers